netbox/ipam/api/serializers_/vlans.py

```diff
@@ -110,3 +112,19 @@ class Meta:
     def validate(self, data):
         # Bypass model validation since we don't have a VID yet
         return data
+
+
+class VLANTranslationRuleSerializer(NetBoxModelSerializer):
+
+    class Meta:
+        model = VLANTranslationRule
+        fields = ['id', 'policy', 'local_vid', 'remote_vid']
+
+
+class VLANTranslationPolicySerializer(NetBoxModelSerializer):
+    rules = VLANTranslationRuleSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = VLANTranslationPolicy
+        fields = ('id', 'url', 'name', 'description', 'display', 'rules')
```

Comment:
I think `fields` needs to be a list (but I'm not certain). It appears to be the convention at least.
```suggestion
        fields = ['id', 'url', 'name', 'description', 'display', 'rules']
```